<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FmrActualizarPaciente</class>
 <widget class="QDialog" name="FmrActualizarPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rbFemenino">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Femenino</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMasculino">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Masculino</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Dni :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>269</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellidos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>269</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDni">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>269</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtDireccion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>69</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefono :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTelefono">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>269</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>91</width>
     <height>88</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>91</width>
     <height>119</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Nacimiento :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFechaNacimiento">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>159</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dosis:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbDosis">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancelar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdActualizar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
